lib/class-sp-integration.php

```diff
@@ -108,10 +108,28 @@ public function remove_hooks() {
 	 * @return array|null
 	 */
 	public function filter__posts_pre_query( $posts, $query ) {
-		if ( ! $query->is_main_query() || ! $query->is_search() ) {
+		$should_auto_integrate = $query->is_main_query() && $query->is_search();
+		if (
+			/**
+			 * Should a query be automatically integrated with SearchPress?
```

Comment:
The [core documentation standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#language) for filters might suggest language that focuses a little more on the "what" of the filter, e.g. `Filters whether a query should automatically be integrated with SearchPress`.

Reply:
Thanks for the suggestion! Updated.